While not an especially important change, every other tool out there uses green
to highlights added lines, not blue.
--- ./html/codestriker.css
+++ ./html/codestriker.css
@@ -21,13 +21,13 @@
 TD.msn {white-space: pre; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: smaller}

 /* Added text */
-TD.a {white-space: pre; background-color: #abd; font-family: Georgia, serif; font-size: smaller}
+TD.a {white-space: pre; background-color: #adb; font-family: Georgia, serif; font-size: smaller}

 /* Added text monospaced */
-TD.msa {white-space: pre; background-color: #abd; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: smaller}
+TD.msa {white-space: pre; background-color: #adb; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: smaller}

 /* Added file */
-TD.af {background-color: #abd}
+TD.af {background-color: #adb}

 /* Added blank text */
 TD.ab {white-space: pre; background-color: #cccccc; font-family: Georgia, serif; font-size: smaller}
@@ -87,7 +87,7 @@
 A.nocom:link:hover { text-decoration: underline; }

 /* Monospaced line with added text */
-SPAN.msa {background-color: #abd; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: medium}
+SPAN.msa {background-color: #adb; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: medium}

 /* Monospaced line with removed text */
 SPAN.msr {background-color: #ff9999; font-family: Consolas, "Nimbus Mono L", LucidaConsole, "Bitstream Vera Sans Mono", monospace; font-size: medium}
